activities.py

```diff
@@ -0,0 +1,62 @@
+# @@@SNIPSTART python-money-transfer-project-template-withdraw
+from temporalio import activity
+
+from shared import PaymentDetails
+from banking_service import BankingService, InvalidAccountError, InsufficientFundsError
+
+# Generic errors for activity failures
+class WithdrawFailedError(Exception):
+    pass
+
+class DepositFailedError(Exception):
+    pass
+
+class RefundFailedError(Exception):
+    pass
+
+@activity.defn
+async def withdraw(data: PaymentDetails) -> str:
+    reference_id: str = f"{data.reference_id}-withdrawal"
+    bank: BankingService = BankingService("bank-api.example.com")
+    try:
+        confirmation: str = bank.withdraw(data.source_account, data.amount, reference_id)
```

Comment:
```suggestion
        return bank.withdraw(data.source_account, data.amount, reference_id)
```

No benefit to the extra line and extra var